src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -368,13 +368,53 @@
+    <!-- keep in sync with src\mono\wasm\wasm.proj -->
+    <ItemGroup>
+      <EmccExportedFunction Include="_malloc" />
+      <EmccExportedFunction Include="_memalign" />
+      <EmccExportedFunction Include="_htons" />
+      <EmccExportedFunction Include="_ntohs" />
+
+      <!-- after 3.0 -->
+      <EmccExportedFunction Include="_memset" Condition="$([MSBuild]::VersionGreaterThan('$(_EmccVersion)', '3.0'))"/>
+
+      <!-- before 3.1.7 see https://github.com/dotnet/runtime/issues/64724 -->
+      <EmccExportedFunction Include="__get_daylight" Condition="$([MSBuild]::VersionLessThan('$(_EmccVersion)', '3.1.7'))"/>
+      <EmccExportedFunction Include="__get_timezone" Condition="$([MSBuild]::VersionLessThan('$(_EmccVersion)', '3.1.7'))"/>
+      <EmccExportedFunction Include="__get_tzname" Condition="$([MSBuild]::VersionLessThan('$(_EmccVersion)', '3.1.7'))"/>
+      <EmccExportedFunction Include="g_free" Condition="$([MSBuild]::VersionLessThan('$(_EmccVersion)', '3.1.7'))" />
+
+      <!-- after 3.1.7 -->
+      <EmccExportedFunction Include="stackSave" Condition="$([MSBuild]::VersionGreaterThanOrEquals('$(_EmccVersion)', '3.1.7'))" />
+      <EmccExportedFunction Include="stackRestore" Condition="$([MSBuild]::VersionGreaterThanOrEquals('$(_EmccVersion)', '3.1.7'))" />
+      <EmccExportedFunction Include="stackAlloc" Condition="$([MSBuild]::VersionGreaterThanOrEquals('$(_EmccVersion)', '3.1.7'))" />
+      <EmccExportedFunction Include="_free" Condition="$([MSBuild]::VersionGreaterThanOrEquals('$(_EmccVersion)', '3.1.7'))"/>
+
+      <EmccExportedRuntimeMethod Include="FS" />
+      <EmccExportedRuntimeMethod Include="print" />
+      <EmccExportedRuntimeMethod Include="ccall" />
+      <EmccExportedRuntimeMethod Include="cwrap" />
+      <EmccExportedRuntimeMethod Include="setValue" />
+      <EmccExportedRuntimeMethod Include="getValue" />
+      <EmccExportedRuntimeMethod Include="UTF8ToString" />
+      <EmccExportedRuntimeMethod Include="UTF8ArrayToString" />
+      <EmccExportedRuntimeMethod Include="FS_createPath" />
+      <EmccExportedRuntimeMethod Include="FS_createDataFile" />
+      <EmccExportedRuntimeMethod Include="removeRunDependency" />
+      <EmccExportedRuntimeMethod Include="addRunDependency" />
+      <EmccExportedRuntimeMethod Include="FS_readFile" />
+    </ItemGroup>
+    <PropertyGroup>
+        <_WasmEHLib Condition="'$(WasmEnableExceptionHandling)' == 'true'">libmono-wasm-eh-wasm.a</_WasmEHLib>
+        <_WasmEHLib Condition="'$(WasmEnableExceptionHandling)' != 'true'">libmono-wasm-eh-js.a</_WasmEHLib>
+        <_WasmEHLibToExclude Condition="'$(WasmEnableExceptionHandling)' == 'true'">libmono-wasm-eh-js.a</_WasmEHLibToExclude>
+        <_WasmEHLibToExclude Condition="'$(WasmEnableExceptionHandling)' != 'true'">libmono-wasm-eh-wasm.a</_WasmEHLibToExclude>
+        <EmccExportedRuntimeMethodsEsc>@(EmccExportedRuntimeMethod -> '%(Identity)', '%27,%27')</EmccExportedRuntimeMethodsEsc>
```

Comment:
Both `EmccExportedRuntimeMethodsEsc` and `EmccExportedRuntimeMethods` are considered private, right?

So they should have prefix `_`.